awx/settings/defaults.py

```diff
@@ -468,6 +468,7 @@
     'send_subsystem_metrics': {'task': 'awx.main.analytics.analytics_tasks.send_subsystem_metrics', 'schedule': timedelta(seconds=20)},
     'cleanup_images': {'task': 'awx.main.tasks.system.cleanup_images_and_files', 'schedule': timedelta(hours=3)},
     'cleanup_host_metrics': {'task': 'awx.main.tasks.system.cleanup_host_metrics', 'schedule': timedelta(days=1)},
+    'host_metric_summary_monthly': {'task': 'awx.main.tasks.system.host_metric_summary_monthly', 'schedule': timedelta(weeks=1)},
```

Comment:
To offer full disclosure, I want to make it clear that the scheduler does not start a schedule on any kind of absolute or invariant time reference.

https://github.com/ansible/awx/blob/aafd4df28876f7b0d3479653a2fb77cb382e2383/awx/main/dispatch/periodic.py#L56

Rephrasing... whenever the dispatcher starts, the `cleanup_host_metrics` job will be set to run 1 day in the future. You can inspect and verify with shell_plus the item `scheduler.jobs[-1].next_run` from that startup loop I link.

My only implication is that when services are restarted, this new `host_metric_summary_monthly` task will be set to run 1 week in the future. This is true for all nodes, as all control plane nodes will run this every week, starting 1 week after their startup.

That's not a blocker for me, I just want you to know and consider if there are any negative consequences for your own goals.

Reply:
Does it mean first run is not immediatelly after restart but after week? 
If so, then I'll shorten the interval and check the interval also in the task (like gather-analytics)

Reply:
I changed this to run the schedule each several hours and check desired interval in the task.